package.json

```diff
@@ -46,6 +46,7 @@
     "docker-compose": "^0.24.1",
     "dockerode": "^3.3.5",
     "get-port": "^5.1.1",
+    "mqtt": "^4.3.7",
```

Comment:
This should be a dev dependency, so it is not there for all testcontainers users. In this way also users can choose whichever mqtt library they like (maybe not useful in this case, but may be for other vendors).

Reply:
Makes total sense, will move it in a second!

Reply:
Moved to `devDependencies`